### PR TITLE
[WIP] Fixed an issue that caused harvest API to download data for only one user

### PR DESCRIPTION
**Pull request by karolkozub**
Look at only the last comment. All the other ones are from PR #72.

### CODE REVIEW

File: GrandCentralBoard.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -781,6 +795,7 @@
 				1AE1981D1CBE8323003B6AB3 /* UIColor.swift */,
 				1A76DFAD1CBBEF8200A2DF49 /* NSBundle.swift */,
 				F928B82D1CC52F77006D0332 /* String+Localized.swift */,
+				F9E920901CC54BF000BC7994 /* NSDate+StringWithFormat.swift */,
```
Comment (ochococo) [inline GrandCentralBoard.xcodeproj/project.pbxproj]: So for every extension to NSDate we will have to add a file?
We could have them grouped by class they extending ex. `NSDate.swift`.

File: GrandCentralBoard/Widgets/Harvest/BillingStatsFetcher.swift [hunk 2]
```diff
@@ -61,7 +61,11 @@ extension NSDate {
     }
 
     func dateDaysAgo(daysAgo: Int) -> NSDate {
-        let timeInterval = NSTimeInterval(-daysAgo * 24 * 3600)
+        return dateWithDayOffset(-daysAgo)
+    }
+
+    func dateWithDayOffset(numberOfDays: Int) -> NSDate {
+        let timeInterval = NSTimeInterval(numberOfDays * 24 * 3600)
```
Comment (ochococo) [inline GrandCentralBoard/Widgets/Harvest/BillingStatsFetcher.swift]: Use `NSDateComponents`.

File: GrandCentralBoard/Widgets/Harvest/BillingUserList.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Decodable
+
+
+typealias BillingUserID = Int
+
+
+struct BillingUserList {
```
Comment (ochococo) [inline GrandCentralBoard/Widgets/Harvest/BillingUserList.swift]: `BillingUsers` ?

**Comment by karolkozub** [comment]
This PR doesn't work without superadmin credentials and requires some additional changes to fix that. Also #72 sort of requires these changes to work correctly, so I'm closing this one and moving the changes to #72 instead.

**Pull request closed without merging**